packages/tasks/src/model-libraries.ts

```diff
@@ -163,6 +163,9 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://fasttext.cc/",
 		snippets: snippets.fasttext,
 		filter: true,
+		countDownloads: {
+            wildcard: { path: "*.bin" },
+        },
```

Comment:
For what I can quickly see on https://huggingface.co/models?library=fasttext&sort=trending, fasttext models are even saved as `model.bin`. Therefore, I would recommend to count downloads on this file specifically. 2 advantages:
- foster consistent naming on the Hub for fasttext models
- exact terms are more efficient than wildcards

```suggestion
		countDownloads: {
			term: { path: "model.bin" },
		},
```

Reply:
Thanks @Wauplin! I agree with you suggestion and have committed it.